src/Controls/src/Core/Handlers/Items/CarouselViewHandler.Windows.cs

```diff
@@ -98,7 +98,15 @@ protected override void OnScrollViewerFound(ScrollViewer scrollViewer)
 			_scrollViewer.ViewChanged += OnScrollViewChanged;
 			_scrollViewer.SizeChanged += OnScrollViewSizeChanged;
 
-			UpdateScrollBarVisibilityForLoop();
+			if (Element.Loop)
+			{
+				UpdateScrollBarVisibilityForLoop();
+			}
+			else
+			{
+				UpdateHorizontalScrollBarVisibility();
```

Comment:
If both `UpdateHorizontalScrollBarVisibility()` and `UpdateVerticalScrollBarVisibility()` are always called together, could consolidate them into a single method.

Reply:
Hi @jsuarezruiz  , I have included the `UpdateScrollBarVisibility()` method to handle both horizontal and vertical updates. Please review it and let me know if you have any feedback or concerns.